apps/teams-test-app/src/components/PagesCurrentAppAPIs.tsx

```diff
@@ -0,0 +1,49 @@
+import { pages } from '@microsoft/teams-js';
+import React, { ReactElement } from 'react';
+
+import { ApiWithoutInput, ApiWithTextInput } from './utils';
+import { ModuleWrapper } from './utils/ModuleWrapper';
+
+const NavigateTo = (): React.ReactElement =>
+  ApiWithTextInput<pages.currentApp.NavigateWithinAppParams>({
+    name: 'navigateTo',
+    title: 'Navigate To',
+    onClick: {
+      validateInput: (input) => {
+        if (!input.pageId) {
+          throw new Error('PageID are required.');
+        }
+      },
+      submit: async (input) => {
+        await pages.currentApp.navigateTo(input);
+        return 'Completed';
+      },
+    },
+  });
+
+const NavigateToDefaultPage = (): React.ReactElement =>
+  ApiWithoutInput({
+    name: 'navigateToDefaultPage',
+    title: 'Navigate To Default Page',
+    onClick: async (setResult) => {
+      await pages.currentApp.navigateToDefaultPage();
+      setResult('Completed');
```

Comment:
no need to call `setResult` if you're returning the same value right after that anyway.

Reply:
Resolved in the latest commit